### PR TITLE
Add TrainingsInfo component

### PR DESCRIPTION
**Pull request by khomtali**
I can't catch this magic of data received from async requests. 😩
When I on `/training` page I have trainings description in `data` variable. When in TrainingInfo cmp `data.map` goes through the array, `data` is still alive. But when I try to read `data.fields.trainingType`, data is getting already `undefined`. 😞
And I'll face the same problem requesting for training plans. Heeelp! 😬🙏🏻

I've also created in this PR one common component Collapsible.

### CODE REVIEW

**Comment by MrTim** [comment]
my best guess is this is happening because of this error
```js
index.js:1 Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    in Main (at App.js:23)
```

Which as far as I can see mean that you are trying to perform a state update on the component which is not there. Which possibly happening because of the loading indicator. Since you do that
```js
  (promiseInProgress === true) ?
      <div id="loading">
        <Heart color="#4bd0a0" />
      </div>
   :
   <>
      <Header />
      <Main />
      <Footer />
   </>
```
Main component would be available and rendered only after you finish performing the action which mean the data would be gone at that point.

What you can do? if this data is not depends on a logged in user then you can preload it on application start and you can easily store in inside a context. This would make application simpler in terms of rerender and should solve the issue.

**Comment by BTBTravis** [comment]
Hey @khomtali still need help on this topic?

**Comment by khomtali** [comment]
@BTBTravis I like Ivan's idea - training type description is actually not so much data, and if you can do the separate endpoint for it (as I mentioned in [#5)](https://github.com/BTBTravis/redi-run-api/issues/5), I can preload it on application start.
But I wouldn't like to do the same with big training plans. And in that case I still have this problem.
What do you think?

**Comment by khomtali** [comment]
I updated current PR with info about training types as a constant. So, of course, the problem with async request is gone, but only for some time :)

File: src/configurations/constants.js [hunk 1]
```diff
@@ -44,4 +44,67 @@ export const zoneDescriptions = [
   },
 ];
 
+export const trainingDescriptions = [
+  {
```
Comment (BTBTravis) [inline src/configurations/constants.js:48]: I personally think hard coding these is not a good idea. My understanding the point of using an external CMS like airtable is to make the training/types data dynamic. So after the app is deployed into a live environment one can update the data with out redeploying the app.

Comment (khomtali) [reply src/configurations/constants.js:48]: Ok, I got you. That's reasonable... But actually training types are kind of constant information. Hope I won't need to change it, only if I'll want to change the training data completely.

**Pull request merged**